backend/cms/views/pages/archive.py

```diff
@@ -4,7 +4,7 @@
 
 from django.contrib.auth.decorators import login_required
 from django.utils.decorators import method_decorator
-from django.shortcuts import render
+from django.shortcuts import render, redirect
```

Comment:
Do we need this import as it seems to be working without it as well?

Reply:
`redirect` is used in [this line](https://github.com/Integreat/cms-django/blob/b3dc29f49535d143712115b394e1ad711d8a5e3b/backend/cms/views/pages/archive.py#L33).

Reply:
But the code worked earlier as well, without the import. I have no problem with leaving it this way. I am just interested why we need the import now, but not earlier

Reply:
If you look at the code, you can see that redirect is only executed if no language parameter is given and the site has a default language. So probably we just never tested this special case.

Reply:
Ahh okay, I see. Thanks pyint :100: :grinning: